.github/workflows/test-package.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        sdk: [stable, dev]
+        sdk: [2.18.0, dev]
```

Comment:
No 2.19.0?

Reply:
We've kinda standardized on oldest + dev. 🤷